app/api/getHost.tsx

```diff
@@ -1,6 +1,6 @@
 import EnvChecker from "../helpers/envChecker";
 
-const DEV_API_HOST = "https://alpha-api.pluto.network"; // TODO: it will be changed after DEV_API deployed
+const DEV_API_HOST = "https://dev-api.pluto.network";
```

Comment:
Can u explain what is the criteria that split the Alpha zone and dev zone in Frontend?